.gitignore

```
@@ -0,0 +1 @@
+/img
```

We need `img` because we have to display images, but we don't need `.vscode` because it's user specific (not everyone use vscode)
```suggestion
.vscode/
```